WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListActivity.kt

```diff
@@ -300,17 +309,102 @@ class PostsListActivity : AppCompatActivity(),
         super.onCreateOptionsMenu(menu)
         menu?.let {
             menuInflater.inflate(R.menu.posts_list_toggle_view_layout, it)
-            val toggleViewLayoutMenuItem = it.findItem(R.id.toggle_post_list_item_layout)
+            toggleViewLayoutMenuItem = it.findItem(R.id.toggle_post_list_item_layout)
             viewModel.viewLayoutTypeMenuUiState.observe(this, Observer { menuUiState ->
                 menuUiState?.let {
                     updateMenuIcon(menuUiState.iconRes, toggleViewLayoutMenuItem)
                     updateMenuTitle(menuUiState.title, toggleViewLayoutMenuItem)
                 }
             })
+
+            searchActionButton = it.findItem(R.id.toggle_post_search)
+
+            viewModel.isSearchAvailable.observe(this, Observer { isAvailable ->
+                if (isAvailable) {
```

Comment:
Checking for `isStarted` doesn't work out of the box for `PostListMainViewModel` (breaks VM) so I assume there will be a little bit more investigative work involved. I think it might be better to handle it as a part of a different issue :)

Reply:
>Checking for isStarted doesn't work out of the box
Ohh, interesting. Let's solve it in another issue as you proposed. Thanks for giving it a try!